x-pack/platform/packages/shared/ai-assistant/ai-assistant-cta/call_to_action.test.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiThemeProvider } from '@elastic/eui';
+import { mountWithIntl } from '@kbn/test-jest-helpers';
```

Comment:
I think this is the only component in this pr that uses enzyme, can we drop new usage of it?

Reply:
Absolutely, great catch.